the-collab-lab/tcl-77-smart-shopping-list
# Bb/bootstrap integration

## Description



We discussed using `bootstrap` and `react-bootstap` in the application this installs them and changes our buttons, inputs, navbar to utilize it. Makes sure the app still looks and functions the same with the `bootstrap` default styling basically.

## Acceptance Criteria


- [x] A working and functioning app that uses the new package 😅

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓    | 💄  Style |

## Updates

### Before


<img width="1512" alt="Screenshot 2024-09-25 at 7 06 50 PM" src="https://github.com/user-attachments/assets/d9bb1c53-1e98-4cf0-ac80-70709b2cd4eb">


### After


<img width="1624" alt="Screenshot 2024-09-25 at 7 06 59 PM" src="https://github.com/user-attachments/assets/25880086-55c5-4672-b36b-6203922a8315">


## Testing Steps / QA Criteria


- Opening the [preview link]() in the PR you should see the changes

To make sure the dependencies and code function as should:
- From your terminal, pull down this branch with `git pull origin bb/bootstrap-integration` and check that branch out with git checkout `bb/bootstrap-integration``
- Then `npm ci` to install the newly added dependencies locally and `npm start` to launch the app.
- Open a second terminal to run the typescript compiler using ` tsc --noEmit --watch` to see any type issue.
- After installing the new dependencies and starting the app you should see that default styling of the `bootstrap` on all buttons and inputs
- Navigate through the application making sure all functionality still works
